File: src/ProjectR/content/scripts/spells/FuzzyOverhaul.lua
function GetTargetType()
    return Single
end

function GetName()
  return "Fuzzy Overhaul"
end

function GetDescription()
  return "Cleanse target from all buffs and debuffs."
end

function IsSupportSpell()
    return true
end

function GetMPCost()
    return .48
end

function GetDelay()
  return .5
end

function GetSpellType()
  return Magical
end

function GetMasteries()
  return {  }
end

function SpellEffect()
  d:RemoveDebuffs()
  ds:RemoveBuffs()
end

File: src/ProjectR.Scripting/content/scripts/spells/FuzzyOverhaul.cs
using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting
{
    public class FuzzyOverhaul : SpellScriptBase
    {
        public override string Name { get { return "Fuzzy Overhaul"; } }
        public override string Description { get { return "Cleanse target from all stat buffs and debuffs."; } }

        public override TargetType TargetType { get { return TargetType.Single; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] { }; } }
        public override SpellType SpellType { get { return SpellType.Magical; } }
        public override bool IsSupportSpell { get { return true; } }

        public override double MPCost { get { return 48; } }
        public override double Delay { get { return .5; } }

        public override void SpellEffect(ICharacter caster, ICharacter target)
        {
            RemoveStatDebuffs();
            RemoveStatBuffs();
        }
    }
}